components/national-team-profile.js

```diff
@@ -0,0 +1,343 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import dynamic from 'next/dynamic';
+
+import { object, string, number } from 'yup';
+import {
+  Grid,
+  Flex,
+  Select,
+  Heading,
+  Text,
+  Switch,
+  Link,
+} from '@chakra-ui/react';
+import { CheckIcon } from '@chakra-ui/icons';
+
+import { buttonVariants } from 'components/prismic-wrapper';
+import Input from 'components/input'; // DO NOT DYNAMIC IMPORT, BREAKS FORMS
+import Textarea from 'components/textarea'; // DO NOT DYNAMIC IMPORT, BREAKS FORMS
+
+const PrismicWrapper = dynamic(() => import('components/prismic-wrapper'));
+const Label = dynamic(() => import('components/label'));
+const Button = dynamic(() => import('components/button'));
+const Container = dynamic(() => import('components/container'));
+const Required = dynamic(() => import('components/required'));
+const InlineError = dynamic(() =>
+  import('components/errors').then(({ InlineError }) => InlineError)
+);
+
+import { api } from 'modules/api';
+import { rem } from 'styles/theme';
+
+const NATIONAL_TEAMS = ['England', 'Scotland', 'Wales'];
+const POSITIONS = ['Keeper/Chaser', 'Beater', 'Seeker'];
```

Comment:
Non-code question, should there be a Utility option for those who genuinely don't have a primary position?

Reply:
I guess this is TBC with coaches, but generally a player should specialise when applying for a national team.